src/components/__tests__/CSSTable.test.tsx

```diff
@@ -0,0 +1,137 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react'
+import { CSSTable } from '../CSSTable'
+
+jest.mock('@patternfly/react-tokens/dist/esm/componentIndex', () => ({
+  test: {
+    '.test-selector': {
+      '--test-property': {
+        name: 'test-property',
+        value: 'test-value',
+      },
+      '--another-property': {
+        name: 'another-property',
+        value: '#123456',
+      },
+    },
+    '.another-selector': {
+      '--list-property': {
+        name: 'list-property',
+        value: 'list-item-1',
+        values: ['list-item-1', 'list-item-2'],
+      },
+    },
+  },
+  'pf-v6-empty': {},
+}))
+
+describe('<CSSTable />', () => {
```

Comment:
Nit: I wouldn't wrap this whole suite in a describe block since it doesn't really clean things up and makes everything indented an extra level.